omnigibson/reward_functions/energy_metric.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+from omnigibson.reward_functions.reward_function_base import BaseRewardFunction
+
+
+class EnergyMetric(BaseRewardFunction):
+    """
+    Energy Metric
+
+    Measures displacement * mass for every link
+
+    Args:
+        measure_work: If true, measure beginning and end delta rather than step by step delta
+    """
+
+    def __init__(self, measure_work=False):
+        # Run super
+        super().__init__()
+        self._reward = 0
+        self.initialized = False
```

Comment:
Remove this. Everywhere this variable is used, check if state_cache == None instead.